ngraph/frontend/paddlepaddle/src/op/slice.cpp

```diff
@@ -0,0 +1,55 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <ngraph/opsets/opset6.hpp>
+#include "slice.hpp"
+#include <paddlepaddle_frontend/utility.hpp>
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+
+NamedOutputs slice (const NodeContext& node) {
+    auto data = node.get_ng_input("Input");
+    auto axes = node.get_attribute<std::vector<int32_t>>("axes");
+    auto starts = node.get_attribute<std::vector<int32_t>>("starts");
+    auto ends = node.get_attribute<std::vector<int32_t>>("ends");
+    auto parialShape = data.get_partial_shape();
+    PDPD_ASSERT(parialShape.is_static(), "slice: must use static shape.");
+    auto shape = parialShape.to_shape();
+    std::vector<int32_t> fixedStarts(shape.size(), 0);
+    std::vector<int32_t> fixedEnds(shape.size(), INT_MAX);
+    int n = 0;
+    for (size_t &&i : axes) {
```

Comment:
minor: no benefit of having 'r-value reference here', can be just "for(auto i : axes)"

Reply:
@luo-cheng2021 Please fix this

Reply:
fixed in 5db9a7978cf77f1c5a972c114be6be4b693a7c2d